pluggable_membrane.go

```diff
@@ -16,14 +16,15 @@ package main
 
 import (
 	"fmt"
-	membrane2 "github.com/nitric-dev/membrane/pkg/membrane"
-	utils2 "github.com/nitric-dev/membrane/pkg/utils"
 	"log"
 	"os"
 	"plugin"
 	"strconv"
 	"strings"
 
+	membrane2 "github.com/nitric-dev/membrane/pkg/membrane"
+	utils2 "github.com/nitric-dev/membrane/pkg/utils"
```

Comment:
the `<packagename>2` imports are hangovers from the pkg refactor, that was done earlier, would you be able to remove these and update references in this file?